<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MixerBar</class>
 <widget class="QWidget" name="MixerBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>34</width>
    <height>18</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item>
      <widget class="QToolButton" name="onButton">
       <property name="minimumSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton:hover
{
	background-color: rgb(0, 255, 127);
	color: white;
	border-width: 2px;
	border-color: black;
}
QToolButton
{
	background-color: rgb(0, 170, 127);
	color: white;
	border-width: 2px;
	border-color: black;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="offButton">
       <property name="minimumSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(130, 0, 65);
color: white;
border-width: 2px;
border-color: black;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
